src/pages/VenueLandingPage/VenueLandingPage.tsx

```diff
@@ -50,6 +53,9 @@ export interface VenueLandingPageProps {
 
 export const VenueLandingPage: React.FunctionComponent<VenueLandingPageProps> = () => {
   const venueId = useVenueId();
+  const { user } = useUser();
+
+  const isLoggedIn = !!user;
   useConnectCurrentVenue();
 
   const venue = useSelector(currentVenueSelectorData);
```

Comment:
Does this include the `venue.id`? If not, can we switch to a different selector that does, then probably wouldn't need to call `useVenueId` above.

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1148/commits/6f64cb54d9d168ab6365655b2e1169886fc48b1e